### PR TITLE
[IA-1801] update swagger

### PR DESCRIPTION
**Pull request by jdcanas**
Good thing we had this, these entries were not very accurate. I copied directly from the encoder and tested the calls myself. 

Some other entries may be out of date, but I tried to clean up what I could think of and what is most relevant to consumers.

I'm not too sure why ListRuntime and ListCluster are so different...

### CODE REVIEW

File: http/src/main/resources/swagger/api-docs.yaml [hunk 1]
```diff
@@ -1582,8 +1583,8 @@ components:
         - scopes
       properties:
         id:
-          type: string
-          description: The internally-referenced ID of the runtime
+          type: long
```
Comment (Qi77Qi) [inline http/src/main/resources/swagger/api-docs.yaml:1586]: 👍

File: http/src/main/resources/swagger/api-docs.yaml [hunk 2]
```diff
@@ -1694,47 +1742,36 @@ components:
...
+        - runtimeConfig
         - status
         - labels
-        - autopauseThreshold
-        - defaultClientId
       properties:
-        internalId:
-          type: string
-          description: Internal resource ID of the runtime
+        id:
```
Comment (Qi77Qi) [inline http/src/main/resources/swagger/api-docs.yaml:1753]: we have both `id` and `internalId`, just want to double check that this is returning `id` but not `internalId`?

**Review by Qi77Qi — COMMENTED** [review]
one question...otherwise lgtm 👍

File: http/src/main/resources/swagger/api-docs.yaml (hunk 2 above)
Comment (jdcanas) [reply http/src/main/resources/swagger/api-docs.yaml:1753]: Yeah, if you trace it, it corresponds to this encoder https://github.com/DataBiosphere/leonardo/blob/develop/http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/api/RuntimeRoutes.scala#L379

verified here:
Screen Shot 2020-04-23 at 5 55 12 PM

**Review by Qi77Qi — APPROVED** [review]

**Pull request merged**